.changeset/witty-wolves-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-form": patch
```

Comment:
Marking as a `patch` change since only the component docs has changed and LabeledTextField can still be used